packages/core/src/components/advanced-select/advanced-select.tsx

```diff
@@ -112,24 +96,16 @@ export class AdvancedSelect implements ComponentInterface {
     this.open = !this.open;
   };
 
-  private close = () => {
```

Comment:
@tfrijsewijk Waarom is deze verwijderd en heb je overal weer expliciet `this.open = false` neergezet? Ziet er namelijk best vies uit om de logica weer op veel plekken terug te zien i.p.v. op 1 plek.